perspective_reddit_bot/moderate_subreddit.py

```diff
@@ -0,0 +1,160 @@
+"""A reddit bot which uses the Perpsective API to help moderate subreddits.
+
+Args:
+      subreddit: (required) A positional argument for the name of the subreddit to moderate.
+      -rule_config_file: (optional) The file which contains moderation rules. Defaults to 'rules.yaml'
+      -output_dir: (optional) If set, scores for all streamed commments are written to a file in this directory.
+
+Copyright 2018 Google LLC
```

Comment:
Sorry, I wasn't detailed enough, copyright info goes into a code comment, rather than in the doc strings. This helps separate what you need to read to understand the code, from the license info. see:   
https://github.com/tensorflow/tensor2tensor/blob/master/tensor2tensor/problems.py